src/components/common/Tag.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <span class="mr-2 text-sm font-medium py-1 px-2 rounded align-middle" :class="[bgColor, computedBgColor, textColor]">
         {{ title }}
-        <i class="fa fa-times pl-1" @click="remove" />
+        <i id="removeTag" class="fa fa-times pl- cursor-pointer" @click="remove" />
```

Comment:
I would leave the padding as it makes it easier to click the cross. Also most sites like github do not use the cursor-pointer for their tag system.
```suggestion
        <i class="fa fa-times pl-1" @click="remove" />
```